openupgradelib/openupgrade.py

```diff
@@ -711,6 +711,18 @@ def rename_fields(env, field_spec, no_deep=False):
             """,
             (new_field, old_field, model),
         )
+        if table_exists(cr, "ir_property"):  # < v18
```

Comment:
I don't use `version_info[0] < 18` because when migrating with OU to v18, you still have `ir_property` table.